src/dashboard/src/pages/Channel/styles.less

```diff
@@ -1,4 +1,4 @@
-@import '~antd/lib/style/themes/default.less';
```

Comment:
I may missed something but why do we need to change the import statement to import from node_modules directly?

Reply:
Should have used the default style in the newly added page